Common/Product/SharedProject/CommonUtils.cs

```diff
@@ -240,28 +180,28 @@ public static string GetAbsoluteDirectoryPath(string root, string relativePath)
         /// either path is invalid.</exception>
         public static string GetAbsoluteFilePath(string root, string relativePath)
         {
-            var rootUri = MakeUri(root, true, UriKind.Absolute, "root");
-            var relUri = MakeUri(relativePath, false, UriKind.RelativeOrAbsolute, "relativePath");
+            var absolutePath = HasStartSeparator(relativePath)
```

Comment:
If it starts with a separator, it's not relative, is it?

Reply:
It's not, its an absolute path from the root of the current drive. That's why it needs to check for at the start.